models/issues/review.go

```diff
@@ -248,9 +249,14 @@ func FindReviews(ctx context.Context, opts FindReviewOptions) ([]*Review, error)
 	if opts.Page > 0 {
 		sess = db.SetSessionPagination(sess, &opts)
 	}
+	distinct := ""
+	if opts.Distinct {
+		distinct = "reviewer_id"
+	}
 	return reviews, sess.
 		Asc("created_unix").
 		Asc("id").
+		Distinct(distinct).
```

Comment:
Not every database support the SQL syntax?

Reply:
Okay, I didn't know this. So using an empty string for `Distinct` is the issue? How about this then:

```
if opts.Distinct {
	return reviews, sess.
		Asc("created_unix").
		Asc("id").
		Distinct("reviewer_id").
		Find(&reviews)
else {
	return reviews, sess.
		Asc("created_unix").
		Asc("id").
		Find(&reviews)
}
```

Reply:
Oh I see there is a test harness for this. I'll work on a test and resubmit.